docs/context/ref/Action.md

```diff
@@ -22,6 +24,17 @@ Accepts an optional `app` parameter in order to specify a specific app.
 
 ## Properties
 
+<details>
+  <summary><code>action</code></summary>
+
+**type**: `string`
+
+The **action** field indicates the type of action:
+- **raiseIntent** :  If no action or **raiseIntent** is specified, then the current default behaviour will be used with `fdc3.raiseIntent` / `fdc3.raiseIntentForContext`
+- **broadcast** : If **broadcast** action is specified, it will call `fdc3.getOrCreateChannel('Channel 1')`to retrieve the channel, then broadcast the context to that channel with `channel.broadcast(context)`. If no, channel has been specified, it will try to broadcast to the current channel (`fdc3.getCurrentChannel()`)
```

Comment:
Same here, use this to replace the shorter description you gave on line 12 of the schema file.